opentelemetry-otlp/Cargo.toml

```diff
@@ -30,7 +30,7 @@ async-trait = "0.1"
 futures-core = "0.3"
 opentelemetry-proto = { version = "0.3", path = "../opentelemetry-proto", default-features = false }
 grpcio = { version = "0.12", optional = true }
-opentelemetry_api = { version = "0.20", default-features = false, path = "../opentelemetry-api" }
+opentelemetry = { version = "0.21", default-features = false, path = "../opentelemetry" }
```

Comment:
nit: we can add this to changelog of OTLP to say something like "Changed dependency from opentelemetry_api to opentelemetry as the latter is now the API crate".

Reply:
Done.